pkg/nfd-topology-updater/nfd-topology-updater.go

```diff
@@ -349,6 +377,9 @@ func (w *nfdTopologyUpdater) updateNRTTopologyManagerInfo(nrt *v1alpha2.NodeReso
 	updateAttributes(&nrt.Attributes, tmAttributes)
 	nrt.TopologyPolicies = deprecatedTopologyPolicies
 
+	attrList := discoverCpuCores()
+	updateAttributes(&nrt.Attributes, attrList)
+
```

Comment:
PTAL https://github.com/kubernetes-sigs/node-feature-discovery/issues/1964#issuecomment-2556886885 
rather than constrain ourselves to the key/value attribute list, I'd explore the option to add new zones describing the E/P split and relationship